test/resources/bluesky-posts-export.csv

```diff
@@ -469,9 +469,9 @@ bafyreigxeyjpv7jhqivrnumkhm7biou3dkx7j25q7x3kceh6myzycmoarq,3lykwtl5no22g,at://d
 
-https://ec.europa.eu/commission/presscorner/detail/en/statement_25_1952",0,1,0,0,,https://bsky.app/profile/eurobot.bsky.social,Eurobot,,https://cdn.bsky.app/img/avatar/plain/did:plc:yw52e2bo6zv7ulryr4ryncpu/bafkreif4pebsptscit4xjnsjbkb2mqlfsglcjo5xlmjdi4dvmgkmysjpxa@jpeg,2023-10-04T22:14:04.497000,1696450444,,,,,,,,,,,,,,,,,,,,,,,,https://ec.europa.eu/commission/presscorner/detail/en/statement_25_1952,ec.europa.eu,https://ec.europa.eu/commission/presscorner/detail/en/statement_25_1952,"Statement by President Macron, Prime Minister Meloni, Chancellor Merz, Prime Minister Tusk, Prime Minister Starmer, President von der Leyen and President Stubb on Peace for Ukraine ahead of President Trump\'s planned meeting with President Putin","We welcome President Trump\'s work to stop the killing in Ukraine, end the Russian Federation\'s war of aggression, and achieve just and lasting peace and security for Ukraine.\nWe are convinced that onl",https://cdn.bsky.app/img/feed_thumbnail/plain/did:plc:yw52e2bo6zv7ulryr4ryncpu/bafkreib7b745vtqsury5774fnmmt444wijx3cdlzkxb7bf6pz6sbjegsmi@jpeg,,,,,,,,,,,,2025-01-01T00:00:00.000000,,1
+https://ec.europa.eu/commission/presscorner/detail/en/statement_25_1952",0,1,0,0,,https://bsky.app/profile/eurobot.bsky.social,Eurobot,,https://cdn.bsky.app/img/avatar/plain/did:plc:yw52e2bo6zv7ulryr4ryncpu/bafkreif4pebsptscit4xjnsjbkb2mqlfsglcjo5xlmjdi4dvmgkmysjpxa@jpeg,2023-10-04T22:14:04.497000,1696450444,,,,,,,,,,,,,,,,,,,,,,,,https://ec.europa.eu/commission/presscorner/detail/en/statement_25_1952|https:\/\/ec.europa.eu\/commission\/presscorner\/detail\/en\/statement_25_1952,ec.europa.eu|https,https:\/\/ec.europa.eu\/commission\/presscorner\/detail\/en\/statement_25_1952,"Statement by President Macron, Prime Minister Meloni, Chancellor Merz, Prime Minister Tusk, Prime Minister Starmer, President von der Leyen and President Stubb on Peace for Ukraine ahead of President Trump\'s planned meeting with President Putin","We welcome President Trump\'s work to stop the killing in Ukraine, end the Russian Federation\'s war of aggression, and achieve just and lasting peace and security for Ukraine.\nWe are convinced that onl",https://cdn.bsky.app/img/feed_thumbnail/plain/did:plc:yw52e2bo6zv7ulryr4ryncpu/bafkreib7b745vtqsury5774fnmmt444wijx3cdlzkxb7bf6pz6sbjegsmi@jpeg,,,,,,,,,,,,2025-01-01T00:00:00.000000,,1
 bafyreiaeixjveoju6jyhfj5ex2owte2sjrqpb3yxzo5mqx66xzeljelwle,3lqfi7mnto52w,at://did:plc:djglyf5epph2tc44lnqsdvfi/app.bsky.feed.post/3lqfi7mnto52w,https://bsky.app/profile/clustz.com/post/3lqfi7mnto52w,1748616664,2025-05-30T16:51:04.213166,did:plc:djglyf5epph2tc44lnqsdvfi,clustz.com,"Emmanuel Macron & his wife's shocking conversation after shoving incident decoded by lip reader, ‘Stay away, you loser’
```

Comment:
the wrong url should probably be ignored if it crashes normalize no? here it generates an absurd domain name `https` and returns non urls in the links column